utilities_help/createToolboxMLTBX.m

```diff
@@ -5,7 +5,8 @@ function createToolboxMLTBX(verNumber)
 % project root.
 % Input:
 %   verNumber- string in major.minor.bugfix format
-prjFileName=fullfile("fsdaToolboxPackaging.prj");
+% Copyright 2023 The MathWorks, Inc.
```

Comment:
Give a line break after the copyright text. Use smart indent on MATLAB to fix the indentation and whitespaces in this file.

Reply:
Added line break and applied smart indentation.